Gemfile

```diff
@@ -92,8 +92,9 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'capybara'
+  gem 'capybara-playwright-driver'
   gem 'cyperful', require: false
-  gem 'selenium-webdriver'
+  gem 'selenium-webdriver' # required by cyperful & dropzone system specs
```

Comment:
Sigh. 🤷🏻‍♂️

Reply:
I think this may have been what I got hung up on before.